incydr/cli/docs/cases.md

```diff
@@ -0,0 +1,34 @@
+# Cases Commands
```

Comment:
We can move these to `incydr/docs/cli` now that the published docs PR was merged.